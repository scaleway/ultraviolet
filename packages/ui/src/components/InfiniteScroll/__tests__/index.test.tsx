import { shouldMatchEmotionSnapshot } from '@utils/test'
import { describe, test } from 'vitest'
import { ActionBar } from '..'

describe('ActionBar', () => {
  test('renders correctly ', () =>
    shouldMatchEmotionSnapshot(<ActionBar>Hello</ActionBar>))

  test('renders correctly with custom rank', () =>
    shouldMatchEmotionSnapshot(<ActionBar rank={2}>I am rank 2</ActionBar>))
})
