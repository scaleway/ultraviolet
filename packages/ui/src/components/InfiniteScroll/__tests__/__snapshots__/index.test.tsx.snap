// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ActionBar > renders correctly  1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  />
</DocumentFragment>
`;

exports[`ActionBar > renders correctly with custom rank 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  />
</DocumentFragment>
`;
